Automattic/wp-calypso
# A4A > Marketplace: Update new hosting page testimonial images

Closes https://github.com/Automattic/jetpack-genesis/issues/469

## Proposed Changes

* This PR updates the testimonial images on the new hosting page.

## Why are these changes being made?

* To update the profile pictures in the testimonial section.

## Testing Instructions

* Open the A4A live link
* Go to Marketplace - Hosting and verify the profile pictures on the testimonial section are updated as shown below. You can compare it with the latest design: 3JELpER0gWwwI1BMrTyGE0-fi-6836_71380

**Standard agency hosting:**

<img width="1415" alt="Screenshot 2024-08-07 at 4 53 18 PM" src="https://github.com/user-attachments/assets/14952c2b-9af9-48d5-9f99-2787377cb147">

**Premier agency hosting:**

<img width="1415" alt="Screenshot 2024-08-07 at 4 53 11 PM" src="https://github.com/user-attachments/assets/13e2f30a-7e32-4b76-8045-bd4b9bdccff4">

**Enterprise agency hosting:**

<img width="1415" alt="Screenshot 2024-08-07 at 4 53 02 PM" src="https://github.com/user-attachments/assets/4f73cded-3b51-4620-b080-3a6d6de226b9">


## Pre-merge Checklist
